Bearded.UI/Controls/Control.cs

```diff
@@ -90,21 +119,17 @@ public void SetFrameNeedsUpdateIfNeeded()
 
         public virtual void SetFrameNeedsUpdate()
         {
-            frameNeedsUpdate = true;
-        }
+            if (Parent == null)
```

Comment:
I think this will cause bugs, since you may call this before adding the control to the hierarchy. In this case, that means it's no harm done: the frame should never be recalculated if the parent is null (because if the parent is null, we'll throw an exception anyway).

Reply:
Do you want me to revert this one? Theoretically one could call this, be lucky and set the parent before the Frame getter is called and not crash otherwise, so this is a logic change.

Reply:
Yes, let's continue to allow this. We can reconsider this logic change if this would reveal a bug.